site/content/en/docs/contribution_guidelines/testing.md

```diff
@@ -120,14 +120,20 @@ and then you can use GUI of the Ginkgo Test Explorer to run individual tests, pr
 You can use the following approach to start up a kind cluster and then run e2e tests from commandline or VSCode,
 attaching them to the existing cluster. For example, suppose you want to test some of the multikueue-e2e tests.
 
-Run `E2E_RUN_ONLY_ENV=true make kind-image-build test-multikueue-e2e` and wait for the `Do you want to cleanup? [Y/n] ` to appear.
+Run `make kind-image-build test-multikueue-e2e E2E_RUN_ONLY_ENV=true` and wait for the `Do you want to cleanup? [Y/n] ` to appear.
 
 The cluster is ready, and now you can run tests from another terminal:
 ```shell
 <your_kueue_path>/bin/ginkgo --json-report ./ginkgo.report -focus "MultiKueue when Creating a multikueue admission check Should run a jobSet on worker if admitted" -r
 ```
 or from VSCode.
 
+You can also rebuild **only the Kueue** image without rebuilding the entire cluster — just run this command from another terminal:
+
+```shell
+make kind-image-build test-multikueue-e2e E2E_RUN_ONLY_KUEUE=true
+```
```

Comment:
@mimowo regarding you comment https://github.com/kubernetes-sigs/kueue/pull/7657#issuecomment-3588082715 we already have it here.

Reply:
yeah. it would also be useful to make clear it works for regular e2e, not just multikueue

Reply:
https://github.com/epam/kubernetes-kueue/blob/80e4c2b49384457723ba43792cbbdce506fc4df1/site/content/en/docs/contribution_guidelines/testing.md?plain=1#L121C41-L121C111

Reply:
Let's make it clear this also works for TAS e2e and regular